plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -178,6 +179,28 @@ protected MaterializedResult getDescribeOrdersResult()
                 .build();
     }
 
+    @Test
+    public void testCreateTableWithPrimaryKey()
```

Comment:
Add test cases that shows the:

1. multiple primary keys
2. nullable columns as primary keys